GeoLib/Polyline.cpp

```diff
@@ -193,6 +193,26 @@ bool Polyline::isClosed() const
 		return false;
 }
 
+bool Polyline::isCoplanar() const
+{
+	std::size_t const n_points (_ply_pnt_ids.size());
+	if (n_points < 4)
+		return true;
+
+	GeoLib::Point const& p0 (*this->getPoint(0));
+	GeoLib::Point const& p1 (*this->getPoint(1));
+	GeoLib::Point const& p2 (*this->getPoint(2));
+	for (std::size_t i=3; i<n_points; ++i)
+	{
+		if (!GeoLib::isCoplanar(p0, p1, p2, *this->getPoint(i)))
+		{
+			ERR ("Point %d is not coplanar to the first three points of the line.", i);
```

Comment:
i prefer debug level here and error level at somewhere calling this function which clearly gives a context that copolanar is bad.